src/commands/codepush/deployment/lib/date-helper.ts

```diff
@@ -0,0 +1,13 @@
+import * as moment from "moment";
+
+export function formatDate(unixOffset: number): string {
```

Comment:
Does this api return unix timestamps instead of ISO formatted strings? I thought we'd agreeded to use ISO 8601 for all dates & times in the api?

How does this logic deal with differing timezones between the server and client?